kebechet/managers/version/version_test.py

```diff
@@ -0,0 +1,83 @@
+"""Tests for version manager."""
+
+import unittest
```

Comment:
In project thoth we adopted `pytest` and `flexmock` for writing and running tests. Could be this test scenario rewritten to those frameworks?

Reply:
Right, ye sure, shouldn't be a problem.